src/data/statuseffects.txt

```diff
@@ -2717,3 +2717,4 @@
 2717	Kinda Damp	blooddrops.gif	1e39ab0a1afcc1c70d5302304f7b93b2	good	none
 2718	Wholesomely Resolved	happy.gif	2b47f4772381abb5ac24e620be92da96	good	none
 2719	Slippery and Speedy	sweaty.gif	defb8cda55243471997f272f928a09f8	neutral	none	cast 1 Drench Yourself in Sweat
+2720	Buzzed on Distillate	chinsweat.gif	d64eab33f648e1a77da23ae516353fb2	neutral	none
```

Comment:
How do we get this? Is this from a use-on-acquire drink or potion or something? Is there a CLI command we can include as the source?

Reply:
It's not a real drink, it's a choice URL.

Gausie said he wanted to make a pseudo-drink that you could pass to `drink`, but if it takes a while longer I might make a `stillsuit` command. No reason we can't have both.

Reply:
That is, in-game you go to your equipped items, and if the stillsuit is on your familiar there's a "distil" link to the right.